src/main/telemetry/telemetry.h

```diff
@@ -53,7 +60,7 @@ typedef struct telemetryConfig_s {
     uint8_t pidValuesAsTelemetry;
     uint8_t report_cell_voltage;
     uint8_t flysky_sensors[IBUS_SENSOR_COUNT];
-    uint8_t smartport_use_extra_sensors;
```

Comment:
This is incompatible change, see PG documentation
https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md